### ISSUE TITLE
Unpin pyparsing==2.4.7 upon new RDFLib release

### ISSUE THREAD

**Issue by jsstevenson** [cancervariants/therapy-normalization#218]
This will probably also affect the Disease Normalizer

With these versions installed, NCIt parsing seems to work:

```
  - owlready2 [required: Any, installed: 0.34]
  - rdflib [required: Any, installed: 6.0.0]
    - isodate [required: Any, installed: 0.6.0]
      - six [required: Any, installed: 1.16.0]
    - pyparsing [required: Any, installed: 2.4.7]
    - setuptools [required: Any, installed: 54.0.0]
```

With these versions installed, NCIt parsing seems to fail:

```
  - owlready2 [required: Any, installed: 0.34]
  - rdflib [required: Any, installed: 6.0.1]
    - isodate [required: Any, installed: 0.6.0]
      - six [required: Any, installed: 1.16.0]
    - pyparsing [required: Any, installed: 3.0.0rc2]
    - setuptools [required: Any, installed: 57.5.0]
```

This is an issue with a change in PyParsing from v2 to v3. The PR to resolve this in RDFLib is merged but hasn't been released yet. We should pin PyParsing==2.4.7 for now and return when the next RDFLib release drops.

**Comment by jsstevenson** [comment]
This is an issue with a change in `PyParsing` from v2 to v3. The PR to resolve this in RDFLib is merged but hasn't been released yet. We should pin PyParsing==2.4.7 for now and return when the next RDFLib release drops.

**Comment by jsstevenson** [comment]
Resolved in #246
